community/x11vnc/build

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh -e
+
+patch -p1 < bugfixes.patch
+patch -p1 < libressl.patch
+
+./autogen.sh
```

Comment:
No way to avoid this (no release tarballs lying around anywhere?)

Reply:
Found a more *recent* tarball from fossies (https://fossies.org/linux/privat/x11vnc-0.9.16.tar.gz/).  Patching still required though.  

Updated bugfixes.patch and source files per latest commit.  Not change to manifest.

Reply:
actually... its the same tarball -.- scratch my previous statement.  still the most recent that i could find.